src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -44,6 +44,28 @@ describe('ReactDOMFiber', () => {
     expect(container.textContent).toEqual('10');
   });
 
+  it('should render dangerouslySetInnerHTML', () => {
```

Comment:
@acdlite OK, fixed it. I've added the tests using `ReactDOM.render` instead of `_createContentMarkup` into `ReactDOMComponent-test`.